.chalice/policy.json

```diff
@@ -18,6 +18,14 @@
         "clouddirectory:*"
       ],
       "Resource": "*"
+    },
+    {
+      "Action": [
+        "secretsmanager:GetSecretValue"
+      ],
+      "Effect": "Allow",
+      "Resource": [
+      ]
```

Comment:
Is this supposed to be empty?

Reply:
yes, it's populated at deployment with key for the secrete in secrets manager.